libvirt.go

```diff
@@ -142,6 +143,52 @@ const (
 	UndefineFlagNVRAM
 )
 
+// Capabilities returns an XML document describing the host's capabilties.
+func (l *Libvirt) Capabilities() ([]byte, error) {
+	resp, err := l.request(constants.ProcConnectGetCapabilties, constants.ProgramRemote, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	r := <-resp
+	if r.Status != StatusOK {
+		return nil, decodeError(r.Payload)
+	}
+
+	dec := xdr.NewDecoder(bytes.NewReader(r.Payload))
+	caps, _, err := dec.DecodeString()
+
+	return []byte(caps), err
+}
+
+// CPUFlags returns a list of CPU flags supported by the host.
+func (l *Libvirt) CPUFlags() ([]string, error) {
```

Comment:
This could easily be left to the consumer as well. Added here because _I_ need it ;) I'd rather keep this library small and simple. How about I just pull `CPUFlags()` if we're going to stick to API-level functionality?